pkg/client/injection/reconciler/autoscaling/v1alpha1/metric/reconciler.go

```diff
@@ -179,7 +179,7 @@ func (r *reconcilerImpl) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
 	// Initialize the reconciler state. This will convert the namespace/name
-	// string into a distinct namespace and name, determin if this instance of
+	// string into a distinct namespace and name, determine if this instance of
```

Comment:
It wouldn't shock me if I'm supposed to change this elsewhere, just let me know where and I should be able to eventually.

Reply:
@tcnghia: there's a `vendor/` directory in this repository, often those are directories that people aren't supposed to change directly, it has instances with `determin` -- can I just change them directly? Or is there an appropriate place to change it. Here's what my local clones have:
https://gist.github.com/jsoref/6115a4ce35488ecc6797ec3cb79563b4

Reply:
don't change vendor directory, it gets auto populated with some tooling we have. is a mix of internal and external modules